api/modules/trips/views.py

```diff
@@ -153,3 +153,30 @@ def remove_friend_from_trip(request, trip_id, user_id):
         return Response(str(e), status=status.HTTP_400_BAD_REQUEST)
 
     return Response(status=status.HTTP_200_OK)
+
+
+@api_view(['POST'])
```

Comment:
Why is this a POST api?
If you want to make it a post API, change the way you take request parameters